pages/vanity-address-generator/index.page.tsx

```diff
@@ -91,7 +91,10 @@ export default function VanityAddressGenerator(): ReactElement {
   const handleChangeCpuCoreCount = (newValue: string): void =>
     handleChangeValidated({
       newValue,
-      validateFn: (newValue) => numberValidator(newValue),
+      validateFn: (newValue) =>
```

Comment:
These changes are not related to this PR. Can you include them in the separate one?